tests/unit/style/properties/serialization.rs

```diff
@@ -1139,3 +1148,26 @@ mod shorthand_serialization {
 
     }
 }
+
+#[test]
+fn test_border_spacing() {
+    use cssparser::Parser;
+    use media_queries::CSSErrorReporterTest;
+    use style::parser::ParserContext;
+    use style::properties::longhands::border_spacing;
+    use style::stylesheets::Origin;
+
+    macro_rules! parse_assert {
+        ($value:expr, $is_exhausted:expr) => {
+            let url = ::servo_url::ServoUrl::parse("http://localhost").unwrap();
+            let context = ParserContext::new(Origin::Author, &url, Box::new(CSSErrorReporterTest));
+            let mut parser = Parser::new($value);
+            let parsed = border_spacing::parse(&context, &mut parser);
```

Comment:
After moving to `mod.rs`, we need to make this macro generic. To do this, we need to replace border_spacing::parse function with `$name::parse` and get the `$name` variable from macro parameters. You can call this macro like this after that: `assert_parser_exhausted!(border_spacing, "1px rubbish", false);`